crates/pop-contracts/src/utils/contracts_node.rs

```diff
@@ -61,6 +62,10 @@ pub async fn run_contracts_node(cache: PathBuf) -> Result<Child, Error> {
 		let tar = GzDecoder::new(file);
 		let mut archive = Archive::new(tar);
 		archive.unpack(cache.clone())?;
+		// Copy the file into the cache folder and remove the folder artifacts
```

Comment:
As mentioned in another, moving the sourcing module to common means some of this code can be eliminated to reuse the functionality exposed there. Having duplicate code is bad for maintenance.

Reply:
Agreed with the refactor, but should be tackled in another PR. This PR is a quick fix to avoid introducing issues in `pop clean` before releasing.